arrow-libs/optics/arrow-optics/src/commonMain/kotlin/arrow/optics/Traversal.kt

```diff
@@ -195,11 +195,11 @@ public interface PTraversal<S, T, A, B> {
       set: (B, B, S) -> T
     ): PTraversal<S, T, A, B> =
       object : PTraversal<S, T, A, B> {
-        override fun <R> foldMap(initial: R, combine: (R, R) -> R, s: S, f: (focus: A) -> R): R =
-          combine(f(get1(s)), f(get2(s)))
+        override fun <R> foldMap(initial: R, combine: (R, R) -> R, source: S, map: (focus: A) -> R): R =
```

Comment:
Sorry, this is a bit nitpicky but do we have a rule of thumb for `map` vs `transform`?